db-client/src/main/java/com/splicemachine/db/client/am/SqlException.java

```diff
@@ -102,7 +102,7 @@
     /** 
      *  The message utility instance we use to find messages
      *  It's primed with the name of the client message bundle so that
-     *  it knows to look there if the message isn't found in the
+     *  ∫it knows to look there if the message isn't found in the
```

Comment:
What's the change for?

Reply:
@jaceklaskowski It's a mistaken key stroke, but no need to correct it.  This PR is not going to merge.  I just opened it to easily share some code for discussion purposes in a different PR.